external/src/test/java/com/redhat/cloud/custompolicies/engine/ReceiverTest.java

```diff
@@ -16,28 +18,31 @@
 import org.hawkular.alerts.api.model.trigger.Trigger;
 import org.hawkular.alerts.api.model.trigger.TriggerAction;
 import org.hawkular.alerts.api.services.DefinitionsService;
+import org.junit.jupiter.api.*;
 import org.junit.jupiter.api.Tag;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestInstance;
 import org.reactivestreams.Publisher;
 
 import javax.inject.Inject;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
 import java.util.*;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
+import static org.junit.jupiter.api.Assertions.*;
 
 @QuarkusTest
 @Tag("integration")
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class ReceiverTest {
 
+    @BeforeAll
+    void init() {
+        System.setProperty("hawkular.data", "./target/hawkular.data");
+    }
+
     @Inject
-    @Channel("kafka-hosts")
-    Emitter<JsonObject> hostEmitter;
+    @Channel("host-egress")
```

Comment:
What about even calling it inventory.host-egress or so? Just to make it a bit more obvious why something incoming is called 'egress'

Reply:
I'm not sure if  dot is allowed there (the properties file could be messed up).